src/components/Payment/PaymentForm/index.tsx

```
@@ -357,67 +361,77 @@ export const PaymentForm = ({ recipient, amount, token, chain, isPintaReq }: Par
-        <div className="space-y-4">
-            {!isPeanutWallet && isWagmiConnected && (
-                <FlowHeader
-                    rightElement={
-                        <Button variant="dark" className="h-7 text-sm" onClick={() => openReownModal()}>
-                            {printableAddress(wagmiAddress!)}
-                        </Button>
+        <div className="space-y-8">
+            <NavHeader
+                title="Send"
+                onPrev={() => {
+                    if (!!user?.user.userId) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)